src/calculateRentalCost.js

```diff
@@ -4,7 +4,20 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let price = days * 40;
+  let smallDiscount = 20;
+  let bigDiscount = 50;
```

Comment:
The 'smallDiscount' and 'bigDiscount' variables should be declared as constants outside of the function since they represent fixed values that do not change.